account_bank_statement_clear_partner/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Account Bank Statement Clear Partner",
```

Comment:
This should be a good title/summary:
```suggestion
    "name": "Clear all partners in bank statement lines",
```

Reply:
Change made